### DISCUSSION TITLE
日後我們還需要學習嗎？

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS3073-C/Python-Community-GS3073-C#677] [General]
當AI科技進化下去，除了少數尖端科學、高難度創作，以及學習和IT科技有關的知識，學其他東西的意義，便不再像現在那麼容易獲肯定：反正都不會勝過AI，人腦的貢獻又有何珍貴之處？大多數情況下，人腦出品都有這樣那樣的缺點和不足，天然的意義和價值何在？科技發展超乎尋常，越來越多人變得可有可無，內在於社會的價值信仰體系，連帶不同專業的社會功能，都會受到前所未見的衝擊。

這篇文章給了我們不少例子與啟發，讓我們思考我們面對大AI時代的來臨還能做些什麼來展示自己的價值，歡迎大家一起思考與討論！

文章連結：https://www.thenewslens.com/article/179434

**Comment by xCk27x** [comment]
我覺得
重要的是做出「**差異化**」 並且善用人類本身的「**創造力**」
在**人類知識的開拓**這一塊 AI仍然沒有辦法取代人類

從 [李開復：人工智慧對人類真正的威脅是什麼？](https://www.inside.com.tw/article/5936-lee-kai-fu-threat-from-ai)
這篇文章中就說到 :
**「人工智慧」機器真正可能帶來的危機，不是奴役人類，而是讓人類喪失鬥志**
想辦法找到每個行業中不可被AI取代的部分
去結合AI這項方便的工具，而不是讓AI取代創造者本身

雖然時間是站在AI這邊沒錯
但AI所帶來的威脅也能使人類進步

**Comment by 0212Jerry** [comment]
我認為人類還是有學習的必要，因為學習是一種社會化的過程，只是在Ai 發展蓬勃的條件下，我們需要學習的東西調整，以往著重於吸收知識的教育逐漸變得無力，我們應該反向思考，與其和Ai 比較，不如多學習Ai 難以學習的東西，例如跨多重領域的整合與應用。

**Comment by Valentinaa0214** [comment]
現今人工智能科技的不斷發展和進化，在某些方面，人工智能甚至可以超越人類的能力，例如大量的數據分析和處理，但人類終究是人類，有著自己的特點，像是獨特的創造力、 思考能力、感知能力等。
因此我認為在人工智能技術日益進步的同時，人類仍然需要不斷學習和提高自己。
人工智能是一種工具，人類需要結合這種工具來更好地思考和創造，並運用人類獨有的創造力和思考能力來處理問題。
